src/nm_mon_daemon.c

```diff
@@ -44,27 +44,31 @@ typedef struct nm_clean_data {
     { NM_MON_VMS_INIT, NULL, NULL, NULL, NM_THR_CTRL_INIT, NM_THR_CTRL_INIT }
 
 #if defined (NM_OS_LINUX)
-static void nm_mon_cleanup(int rc, void *arg)
+static nm_clean_data_t *clean_ptr = NULL;
```

Comment:
I'm not sure if this should be out of #ifdef, or all code, using it should be in #ifdefs. @0x501D

Reply:
maybe this?

https://github.com/nemuTUI/nemu/pull/89/commits/f9f99dd245ab9934ac0b86707b4dc9cbe3cfa8d6

Reply:
Remove this define. It was used for on_exit(3) call